lib/events-metadata.accessor.ts

```diff
@@ -10,6 +10,15 @@ export class EventsMetadataAccessor {
   getEventHandlerMetadata(
     target: Type<unknown>,
   ): OnEventMetadata[] | undefined {
+    // Circumvent a crash that comes from reflect-metadata if it is
+    // given a non-object non-function target to reflect upon.
+    if (
+      target === undefined ||
+      (typeof target !== 'function' && typeof target !== 'object')
```

Comment:
can `target` be `null` somehow? why not:
```suggestion
      !target ||
      (typeof target !== 'function' && typeof target !== 'object')
```

Reply:
Quite right - I misinterpreted `IsObject` thinking that `null` would be handled gracefully, but it won't be. Fixing.

Reply:
(The function that determines what throws is `IsObject` inside of `Reflect.js` in the reflect-metadata npm package)

Reply:
Updated with suggested change